query_optimizer/resolver/NameResolver.hpp

```diff
@@ -47,8 +49,12 @@ class NameResolver {
  public:
   /**
    * @brief Constructor.
+   *
+   * @param parent_resolver The NameResolver inherited from the outer query.
+   *                        NULL if there is no outer query.
```

Comment:
FYI, as a convention, this line should align to the first letter of `parent_resolver`. See below for the more details:

```
  * @param parent_resolver The NameResolver inherited from the outer query.
  *        NULL if there is no outer query.
```